src/fusor/harvester.py

```diff
@@ -162,3 +166,29 @@ class GenieHarvester(FusionCallerHarvester):
     }
     delimeter = "\t"
     fusion_caller = Genie
+
+
+class CIVICHarvester(BaseModel):
+    """Class for harvesting CIViC Fusion objects"""
+
+    fusions_list: list[FusionVariant]
+    model_config = ConfigDict(arbitrary_types_allowed=True)
+
+    def load_records(self) -> list[CIVIC]:
+        """Extract data from CIVIC fusion objects
+
+        :return A list of CIVIC objects
+        """
+        params = {}
+        processed_fusions = []
+        for fusion in self.fusions_list:
+            params["vicc_compliant_name"] = fusion.vicc_compliant_name
+            params["five_prime_end_exon_coordinates"] = (
+                fusion.five_prime_end_exon_coordinates
+            )
+            params["three_prime_start_exon_coordinates"] = (
+                fusion.three_prime_start_exon_coordinates
+            )
+            params["molecular_profiles"] = fusion.molecular_profiles
+            processed_fusions.append(CIVIC(**params))
+        return processed_fusions
```

Comment:
You could also consider using `update`. 

```suggestion
        processed_fusions = []
        for fusion in self.fusions_list:
            params = {
                "vicc_compliant_name": fusion.vicc_compliant_name,
                "five_prime_end_exon_coordinates": fusion.five_prime_end_exon_coordinates,
                "three_prime_start_exon_coordinates": fusion.three_prime_start_exon_coordinates,
                "molecular_profiles": fusion.molecular_profiles
            }
            processed_fusions.append(CIVIC(**params))
        return processed_fusions
```